.gitlab-ci.yml

```diff
@@ -43,21 +43,23 @@ variables:
 
 ##### MERGE REQUEST PIPELINE #####
 
-build_unit_test_job:
+build_job:
   extends:
-    - .build_unit_tests
+    - .build
   stage: build
 
+unit_test_job:
+  stage: unit_test_coverage
+  extends:
+    -  .unit_test
```

Comment:
after combining unit and coverage we can extend the same here: .unit_test_and_coverage